misc/python/materialize/buildkite.py

```diff
@@ -201,6 +203,23 @@ def _upload_shard_info_metadata(items: list[str]) -> None:
     )
 
 
+def add_failure_for_qa_team(failure: str) -> None:
```

Comment:
Optional: `notify_qa_team_on_slack_about_failure`?

Reply:
I think using Slack is an implementation detail, we could also switch to some other mechanism.